### PR TITLE
landing page, navbar updated. footer created. 

### PR DESCRIPTION
**Pull request by KenMain5**
Changes: 
1. Tailwindconfig: Previously, all the pages were dark although I specified it to be a specific color. Changed so that it would be specified colors now 

2. Completed Navbar 
navbar

3. Buttons will appear once the user has logged in
one
two

4. Completed Footer
footer

Let me know if the Navbar Links in the Navbar.jsx needs to be moved in a different file.

### CODE REVIEW

File: src/components/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import React from 'react'
```
Comment (clee4037) [inline src/components/Navbar.jsx:1]: nice work with the nav bar

File: src/components/footer/FooterEntry.jsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+function FooterEntry({ entry }) {
+  return (
+    <div className='flex flex-col gap-2'>
+        {Object.entries(entry).map(([key, value]) => (
```
Comment (clee4037) [inline src/components/footer/FooterEntry.jsx:4]: Damn I actually never knew you could do this. I've always just for looped through the object

**Review by clee4037 — APPROVED** [review]
Good job splitting up components!

**Review by jose-m-f — APPROVED** [review]
Hey Kenny, everything here looks great. Thanks for the work you did with all the refactoring.

**Pull request merged**